zqd/ingest/log.go

```diff
@@ -123,7 +123,9 @@ func ingestLogs(ctx context.Context, pipe *api.JSONPipe, s *space.Space, req api
 		return err
 	}
 	if tailW.r != nil {
-		if err = s.SetTimes(tailW.r.Ts, headW.r.Ts); err != nil {
+		min := nano.Min(tailW.r.Ts, headW.r.Ts)
+		max := nano.Max(tailW.r.Ts, headW.r.Ts)
+		if err = s.SetSpan(nano.NewSpanTs(min, max)); err != nil {
```

Comment:
This span is incorrect as spans are open on the right... I think you need to make it `NewSpanTs(min, max+1)` ?

Reply:
I think you are right.